MediaProcessor/src/FFmpegCommandBuilder.h

```diff
@@ -31,70 +39,115 @@ class FFmpegCommandBuilder : public CommandBuilder {
     FFmpegCommandBuilder& addOverwrite();
 
     /**
-     * @brief Adds input flag and input file path
+     * @brief Adds input flag and the input file path from FFmpegSettingsManager
```

Comment:
Perhaps a find-replace for FFmpegSettingsManager as it's now the FFmpegConfigManager

Reply:
Good eyes, that was the last remaining reference to the old settings manager